ios/Exponent/Versioned/Core/UniversalModules/EXScopedScreenOrientation.h

```diff
@@ -0,0 +1,15 @@
+// Copyright © 2018-present 650 Industries. All rights reserved.
+
+
```

Comment:
We will need to wrap definitions of scoped modules with `#if __has_include(<EXScreenOrientation/EXScreenOrientationModule.h>)`